src/components/inputs/text/ValidationInput/ValidationInput.tsx

```diff
@@ -34,16 +40,22 @@ function ValidationInput({
   disabled?: boolean; // disables input
   value: string | number | undefined;
   setValue: (text: string) => void;
-  setIsValid?: (validity: boolean) => void;
+  validityCallback?: (validity: boolean) => void;
   validationPredicates: { [x: string]: (value: string) => Promise<any> };
   onClick?: () => void;
 }) {
   const [validationResults, setValidationResults] =
     useState<ValidationObject[]>();
 
+  const [valid, setValid] = useState<undefined | boolean>(undefined);
+  const [openTooltip, setOpenTooltip] = useState(false);
+
   useEffect(() => {
     const inputEle = document.getElementById(inputId);
     inputEle?.focus();
+    if (value) {
+      validationExecutor(value.toString());
+    }
   }, [disabled]);
```

Comment:
should we be listening for changes to `value` here?

Reply:
> 

Updated validation input tooltip to recquire validations to open tooltip